pkg/apis/machine/v1alpha1/register.go

```diff
@@ -20,8 +20,28 @@ var (
 	AddToScheme = SchemeBuilder.AddToScheme
 )
 
-// GroupName is the group name use in this package
-const GroupName = "machine.sapcloud.io"
+const (
+	// GroupName is the group name use in this package
+	GroupName = "machine.sapcloud.io"
+
+	// AnnotationKeyMachineUpdateFailedReason is the annotation key for the machine indicating the reason for the machine update failure.
+	AnnotationKeyMachineUpdateFailedReason = "node.machine.sapcloud.io/update-failed-reason"
+
+	// LabelKeyMachineCandidateForUpdate is the label key for the machine indicating the machine will undergo an update.
+	LabelKeyMachineCandidateForUpdate = "node.machine.sapcloud.io/candidate-for-update"
+	// LabelKeyMachineSelectedForUpdate is the label key for the machine indicating the machine is selected for update.
+	LabelKeyMachineSelectedForUpdate = "node.machine.sapcloud.io/selected-for-update"
+	// LabelKeyMachineReadyForUpdate is the label key for the machine indicating the machine is ready for update after the node drain.
+	LabelKeyMachineReadyForUpdate = "node.machine.sapcloud.io/ready-for-update"
+	// LabelKeyMachineDrainSuccessful is the label key for the machine indicating the node drain was successful.
+	LabelKeyMachineDrainSuccessful = "node.machine.sapcloud.io/drain-successful"
+	// LabelKeyMachineUpdateSuccessful is the label key for the machine indicating the machine update was successful.
+	LabelKeyMachineUpdateSuccessful = "node.machine.sapcloud.io/update-successful"
+	// LabelKeyMachineUpdateFailed is the label key for the machine indicating the machine update failed.
+	LabelKeyMachineUpdateFailed = "node.machine.sapcloud.io/update-failed"
+	// LabelKeyMachineSetSkipUpdate is the label key for the machine indicating the machine set update should be skipped.
+	LabelKeyMachineSetSkipUpdate = "node.machine.sapcloud.io/machine-set-skip-update"
+)
```

Comment:
Are you sure this is the right file for these constants?
To me `machine_types.go` seems more apt, or even a new file `constants.go`

Reply:
Addressed both comments.
PTAL.